### DISCUSSION TITLE
Custom labelmap.txt with HA addon/docker?

### DISCUSSION THREAD

**Discussion by iMiMx** [blakeblackshear/frigate#1093] [Ask A Question]
Currently using the HA addon, is there a way to specify the custom labelmap when using the HA addon?

https://blakeblackshear.github.io/frigate/configuration/objects#customizing-the-labelmap

The above specifies:

"Mount the new file at /labelmap.txt in the container with an additional volume"

I'm assuming just putting it in /config on a supervised install isn't enough?  Or is it?  I did try this, saw no reference in the start-up logs - assuming there would be, of course.

**Comment by iMiMx** [comment]
Am testing dropping the modified labelmap.txt into /config on hassio supervised install - I've merged cats into dogs, so there are 2 dog entries, I'm not seeing any cats detected so far in test...

Not sure how else to determine whether or not this has any impact, guess we shall see...

**Comment by blakeblackshear** [comment]
There is no way to do this with the addon currently.

**Reply by iMiMx** [reply]
Thanks for the reply - guess my small terrier will just have to put up with being labelled a cat!

**Reply by jowgn** [reply]
Are there any plans to implement it? Having the same problem, just the opposite way. Our cat is called dog😄
